packages/verkle/src/types.ts

```diff
@@ -142,3 +143,6 @@ export const HEADER_STORAGE_OFFSET = 64
 export const CODE_OFFSET = 128
 export const VERKLE_NODE_WIDTH = 256
 export const MAIN_STORAGE_OFFSET = BigInt(256) ** BigInt(31)
+
+export const zeroValues = new Array(256).fill(new Uint8Array())
+export const zeroCValues = new Array(128).fill(new Uint8Array())
```

Comment:
To me "CValue" is a bit unclear. Do you think it would be clearer if we named this "zeroLeafValues"?

Reply:
Not really sure.  It's not precisely `zeroLeafValues` but more like the `zeroHalfLeafValues`.  What do you think?

Reply:
Hmm yeah. I'm not convinced about what the naming should actually be, but I don't find zeroValues/zeroCValues insightful/clear either. I'll try to think of alternatives..

Reply:
Maybe just 
```
export const emptyByteArrays = new Array(256).fill(new Uint8Array());
export const emptyByteArraysHalfSize = new Array(128).fill(new Uint8Array());
```
 ?

Reply:
Okay, lets go with this for now. Will update.

Reply:
On second thought, I don't see these used anywhere so no real point I don't think. I'm just going to remove entirely until we actually use them.

Reply:
Fair enough!